docs/resources/apig_custom_authorizer.md

```diff
@@ -85,13 +86,14 @@ The `identity` block supports:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - ID of the custom authorizer.
-* `create_time` - Time when the APIG custom authorizer was created.
+
+* `created_at` - The creation time of the custom authorizer.
 
 ## Import
 
-Custom Authorizers of the APIG can be imported using their `name` and the ID of the APIG instance to which the group belongs,
-separated by a slash, e.g.
+Custom Authorizers of the APIG can be imported using their `name` and related dedicated instance IDs, separated by a
```

Comment:
the desc should keep same order with import

Reply:
The mapping relationship of description and the import arguments is not easy to confuse and doesn't need to be modified.